docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,313 @@
+
+1. Create a `Library` folder under `FSNetCore/src`.
+2. In the command line, execute `dotnet new -l F#` in `FSNetCore/src/Library`.
+3. Remove the `NuGet.Config` file.
+4. Rename `Program.fs` to `Lib.fs`.
+5. Open the `project.json` file and remove the `emitEntryPoint` entry from `buildOptions`.
+6. Under `buildOptions/compile/includeFiles`, replace `Program.fs` with `Lib.fs`.
+7. Remove the global `dependencies` section.
+8. Under `frameworks`, change `netcoreapp1.0` to `netstandard1.6`.
+9. Under `frameworks/netstandard1.6`, remove the `imports` section.
+10. Under `frameworks/netstandard1.6/dependencies`, replace the `Microsoft.NETCore.App` package with `"NETStandard.Library":"1.6.0"`.  Add `"Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629"` and `"Newtonsoft.Json": "9.0.1-beta1"`.
+11. Open `Lib.fs` and change the contents to the following code:
+    ```fsharp
+    module Library
+
+    open Newtonsoft.Json
+
+    let getJsonNetJson value = 
+        sprintf "I used to be %s but now I'm %A!" <|| (value,  JsonConvert.SerializeObject(value))
+    ```
+12. Run `dotnet restore` and `dotnet build`.  These should succeed.
+
+Your `project.json` file should look like this:
+
+```json
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Lib.fs"
+      ]
+    }
+  },
+  "tools": {
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-preview2-*",
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win81",
+        "netstandard1.3"
+      ]
+    }
+  },
+  "frameworks": {
+    "netstandard1.6": {
+      "dependencies": {
+        "NETStandard.Library":"1.6.0",
+        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+        "Newtonsoft.Json": "9.0.1-beta1"    
+      }
+    }
+  }
+}
+```
+
+And your `Lib.fs` file should look like this:
+
+```fsharp
+module Library
+
+open Newtonsoft.Json
+
+let getJsonNetJson value = 
+    sprintf "I used to be %s but now I'm %A!" <|| (value,  JsonConvert.SerializeObject(value))
+```
+
+### Writing a Console Application which Consumes the Class Library
+
+1. Create an `App` folder under `FSNetCore/src`.
+2. In the command line, execute `dotnet new -l F#` in `FSNetCore/src/App`.
+3. Remove the `NuGet.Config` file.
+4. Open the `project.json` file.
+5. Remove the global `dependencies` section.
+6. Under `frameworks/netcoreapp1.0/`, remove the `imports` section.
+7. Under `frameworks/netcoreapp1.0/dependencies`, add the following after `Microsoft.NETCore.App`:
+    ```json
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+    "Library":{
+      "version":"1.0.0",
+      "target": "project"
+    }
+    ```
+6. Change `Program.fs` to:
+    ```fs
+    open System
+    open Library
+
+    [<EntryPoint>]
+    let main argv = 
+        printfn "Nice command line arguments!  Here's what JSON.NET has to say about them:\n"
+
+        for arg in argv do
+            printfn "%s" <| getJsonNetJson arg
```

Comment:
maybe `|>` it's more idiomatic

```fsharp
arg |> getJsonNetJson |> printfn "%s"
```

or 

```fsharp
printfn "%s" (getJsonNetJson arg)
```

also, condider user `Array.iter` instead of `for`?

```fsharp
argv
|> Array.map getJsonNetJson
|> Array.iter (printfn "%s")
```